api/clients/v2/payloaddispersal/payload_disperser.go

```diff
@@ -68,16 +71,20 @@ func NewPayloadDisperser(
 //  4. Construct an EigenDACert if dispersal is successful
 //  5. Verify the constructed cert with an eth_call to the EigenDACertVerifier contract
 //  6. Return the valid cert
+// TODO(ethen): Figure out the best interface for this function. Is versioning the return type acceptable
+//              or should we propagate bytes instead?
 func (pd *PayloadDisperser) SendPayload(
 	ctx context.Context,
 	// payload is the raw data to be stored on eigenDA
 	payload *coretypes.Payload,
-) (*coretypes.EigenDACert, error) {
+) (coretypes.EigenDACert, error) {
 
 	probe := pd.stageTimer.NewSequence()
 	defer probe.End()
 	probe.SetStage("convert_to_blob")
 
+	// convert the higher level payload into a low level EigenDA blob using the polynomial form
+	// to dictate whether the representation is in coefficient (requires IFFT) or evaluation form
```

Comment:
wouldn't use higher/lower nomenclature here. its just an encoding

Reply:
also I feel like "to dictate ..." is confusing. We're just specifying what polynomial form to interpret the (untyped) payload as.

Actually for comments like this I'm a bit torn.. given that these kinds of comments are only useful for people reading the code in github. Anyuone with an IDE should instead read the comment on the function ToBlob() itself. So should we really add comments like this that are just duplicated and potentially more confusing if they don't say the same thing as the ToBlob() function?

Reply:
```suggestion
	// convert the payload into an EigenDA blob by interpreting the payload in polynomial form,
	// which means the encoded payload will need to be IFFT'd since EigenDA blobs are in coefficient form.
```